### PR TITLE
Display gravatar image if user has our login

### PR DESCRIPTION
**Pull request by domenc**
Fix user picture if has our login (not FB login) - display gravatar image

### CODE REVIEW

File: piplmesh/frontend/templates/gravatar.html [hunk 1]
```diff
@@ -1 +1,2 @@
-<img src="{{ gravatar.url }}" width="{{ gravatar.size }}" height="{{ gravatar.size }}" class="gravatar" />
\ No newline at end of file
+{% load i18n %}
+<li class="image"><img src="{{ gravatar.url }}" height="{{ gravatar.size }}" width="{{ gravatar.size }}" alt="{% trans "User picture" %}" /></li>
```
Comment (mitar) [inline piplmesh/frontend/templates/gravatar.html]: No, `li` should not be in this template. We can use the same tag in multiple places. Only `img` should be there.

File: piplmesh/frontend/templates/header.html [hunk 2]
```diff
@@ -30,7 +30,8 @@
                     {% if user.facebook_id %}
                         <li class="image"><img src="{% facebook_picture user.facebook_id "square" %}" alt="{% trans "User picture" %}" /></li>
                     {% else %}
-                        <li class="image"><img src="{{ STATIC_URL }}piplmesh/images/unknown.png" height="50" width="50" alt="{% trans "User picture" %}" /></li>
+                        {% load gravatar %}
```
Comment (mitar) [inline piplmesh/frontend/templates/header.html]: Load tag libraries always all together at the top of the template.

File: piplmesh/frontend/templates/gravatar.html (hunk 1 above)
Comment (domenc) [inline piplmesh/frontend/templates/gravatar.html]: OK

File: piplmesh/frontend/templates/header.html (hunk 2 above)
Comment (domenc) [inline piplmesh/frontend/templates/header.html]: OK

**Pull request merged**